index.js

```diff
@@ -10,7 +10,7 @@ async function main() {
     console.log(chalk.blue('🚀 Welcome to Todo CLI!'));
     console.log(chalk.blue('='.repeat(50)));
     
-    while (true) {
```

Comment:
Changed condition in the while loop to 'true === true' for explicit comparison, which may not be necessary.